### ISSUE TITLE
canonical-certification-precheck script is warning of the absence of a [common] config file section

### ISSUE THREAD

**Issue by rodwsmith** [canonical/checkbox#2489]
### Bug Description

The `canonical-certification-precheck` script used in Server Certification is checking for the presence of a `[common]` section in `/etc/xdg/canonical-certification.conf`; however, this configuration file does not currently include such a section. The result is that the `XDG_Check` precheck script test fails. Either the default config file should be updated to include such a section or the precheck script should be modified to omit this check, so as to avoid confusing users.

### Cert-blocker Test Case

- [ ] cert-blocker

### To Reproduce

1. Install the `canonical-certification-server` package.
2. Run the `canonical-certification-precheck` script.
3. Answer various questions in the script and observe the output.

### Expected Result

The `XDG_Check` test should pass (assuming the config file is otherwise OK).

### Actual Result

The `XDG_Check` test fails.

### Environment

- Tested with both Ubuntu 24.04 and 25.10
- `canonical-certification-server` 7.0.0~dev150~ubuntu24.04.1 and 7.0.0~dev150~ubuntu25.10.1

### Relevant log output

```shell
XDG_Check  --------------------  Failed
```

**Comment by Hook25** [comment]
Hello, if that file is supposed to be a checkbox config, the test needs updating indeed, there is no such section

**Comment by Hook25** [comment]
closing this due to complete inactivity and impossibility to fix this in its current state
